media-libs/osl/osl-1.10.5.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils toolchain-funcs
+
+DESCRIPTION="Advanced shading language for production GI renderers"
+HOMEPAGE="http://opensource.imageworks.com/?p=osl"
+SRC_URI="https://github.com/imageworks/OpenShadingLanguage/archive/Release-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+X86_CPU_FEATURES=(
+	sse2:sse2 sse3:sse3 ssse3:ssse3 sse4_1:sse4.1 sse4_2:sse4.2
+	avx:avx avx2:avx2 avx512f:avx512f f16c:f16c
+)
+CPU_FEATURES=( ${X86_CPU_FEATURES[@]/#/cpu_flags_x86_} )
+
+IUSE="doc partio qt5 test ${CPU_FEATURES[@]%:*}"
+
+RDEPEND="
+	dev-libs/pugixml
+	>=media-libs/openexr-2.2.0:=
+	>=media-libs/openimageio-1.8.5
+	sys-libs/zlib:=
+	partio? ( media-libs/partio )
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+	)
+"
+
+DEPEND="${RDEPEND}"
+BDEPEND="
+	>=dev-libs/boost-1.62
+	sys-devel/clang
```

Comment:
and same for `sys-devel/clang`

Reply:
Isn't clang as a compiler being considered part of a toolchain and therefore needed to put into BDEPEND?

Reply:
package is depending on `sys-devel/clang` for headers, not the compiler, right? I'm quite sure it should use the system compiler instead, GCC in my case.

Reply:
probably, some targets also link against llvm libraries, but you're right, in this case it's not part of the actual toolchain.